tooling-tests/src/test/resources/test-src/samples/tool_test_init_2/Config.toml

```diff
@@ -0,0 +1 @@
+foo1 = "bar1"
```

Comment:
```suggestion
foo1 = "bar1"

```

Reply:
Updated in [24eda25](https://github.com/ballerina-platform/persist-tools/pull/7/commits/24eda2553c3805131632ee9181c97b4242370b18)